great_expectations/cli/init.py

```diff
@@ -76,6 +76,8 @@ def init(target_directory, view):
             sys.exit(1)
         else:
             try:
+                cli_message("Target directory: " + target_directory)
+                sys.exit(0)
```

Comment:
These two lines were probably left here unintentionally and are breaking integration tests.

Reply:
You're the best, thanks for spotting.